x-pack/plugins/fleet/cypress/plugins/index.ts

```diff
@@ -5,12 +5,42 @@
  * 2.0.
  */
 
+import { promisify } from 'util';
+
+import fs from 'fs';
+
+import fetch from 'node-fetch';
 import { createEsClientForTesting } from '@kbn/test';
 
 const plugin: Cypress.PluginConfig = (on, config) => {
   const client = createEsClientForTesting({
     esUrl: config.env.ELASTICSEARCH_URL,
   });
+
+  async function kibanaFetch(opts: {
+    method: string;
+    path: string;
+    body?: any;
+    contentType?: string;
+  }) {
+    const { method, path, body, contentType } = opts;
+    const Authorization = `Basic ${Buffer.from(
+      `elastic:${config.env.ELASTICSEARCH_PASSWORD}`
```

Comment:
Does anyone know how to add an env var with the `elastic` user in? We currently only have one with system_indices_superuser I can't see how they are set!

Reply:
Have you tried `config.env.ELASTICSEARCH_USERNAME` ?